docs/content/sdks.mdx

```diff
@@ -0,0 +1,54 @@
+# SDKs
+
+import TrackingLink from "./components/tracking-link";
+
+THe following is a list of SDK;s from the FLow community that are currently in development.
+
+Legend:
+
+- 😀: Activity in the last six months
+- 📘: Documentation
+
+Want your SDK listed here? Please fork the [`flow` repository](https://github.com/onflow/flow), edit [this page](https://github.com/onflow/flow/blob/master/docs/content/sdks.md), and submit a pull request.
+
+## JVM (Java, Kotlin, Scala, etc.)
```

Comment:
Why were the Go and JS SDKs removed, but the JVM one kept? If it's only community ones, maybe remove it too?

Reply:
JVM in in house?

Reply:
LOL, no idea what to make of this